selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/Consoles.java

```diff
@@ -409,6 +411,11 @@ public void waitExpectedTextIntoConsole(String expectedText) {
     updateProjDriverWait.until(textToBePresentInElement(consoleContainer, expectedText));
   }
 
+  /** wait on staring jdt.ls */
+  public void waitUntilJdtLsStarted() {
```

Comment:
To be more precise, we are waiting on some message appearance only: `waitJDTLSStartedMessage`

Reply:
As you wish, i will fix it